process/block/metablock.go

```diff
@@ -1003,6 +1003,8 @@ func (mp *metaProcessor) CommitBlock(
 
 	mp.commitEpochStart(header, body)
 
+	mp.validatorStatisticsProcessor.DisplayRatings(header.GetEpoch())
```

Comment:
This would display the ratings after each commit.
Shouldn't we only display after epoch start?

Reply:
yes, moved from UpdatePeerState that was done evey block to commitblock, that is done every block.